ui/pages/confirmations/components/confirm/info/hooks/useTokenDetails.ts

```diff
@@ -1,20 +1,27 @@
 import { TokenListMap } from '@metamask/assets-controllers';
 import { TransactionMeta } from '@metamask/transaction-controller';
 import { useSelector } from 'react-redux';
+import { useI18nContext } from '../../../../../../hooks/useI18nContext';
 import { getTokenList } from '../../../../../../selectors';
 import { SelectedToken } from '../shared/selected-token';
 
-export const useTokenImage = (
+export const useTokenDetails = (
   transactionMeta: TransactionMeta,
   selectedToken: SelectedToken,
 ) => {
+  const t = useI18nContext();
+
   const tokenList = useSelector(getTokenList) as TokenListMap;
```

Comment:
One for future note but `getTokenList` is derived from the global network where as I just added a selector to use `tokensChainsCache` which is multi-chain compatible as it's a map of data per chain.